x-pack/plugins/security_solution/scripts/run_cypress/utils.ts

```diff
@@ -59,7 +59,12 @@ export const isSkipped = (filePath: string): boolean => {
   return callExpression?.callee?.property?.name === 'skip';
 };
 
-export const parseTestFileConfig = (filePath: string): SecuritySolutionDescribeBlockFtrConfig => {
+export const parseTestFileConfig = (
+  filePath: string
+): {
+  ftrConfig: SecuritySolutionDescribeBlockFtrConfig;
+  devConfig: boolean;
```

Comment:
nit: Consider naming this to something that answers a yes/no. `devConfig` seems like a collection/object but it is a `boolean`. `isDevRun`/`isDevMode` or something like that?